crates/modelardb_storage/src/parser.rs

```diff
@@ -47,11 +49,117 @@ use sqlparser::tokenizer::Token;
 use crate::error::{ModelarDbStorageError, Result};
 use crate::metadata::model_table_metadata::{GeneratedColumn, ModelTableMetadata};
 
-/// Constant specifying that a model table should be created.
-pub const CREATE_MODEL_TABLE_ENGINE: &str = "ModelTable";
+/// A top-level statement (SELECT, INSERT, CREATE, UPDATE, etc.) that have been tokenized, parsed,
+/// and for which semantics checks have verified that it is compatible with ModelarDB. CREATE TABLE
+/// and CREATE MODEL TABLE are supported.
```

Comment:
Maybe add more or remove the last sentence since it makes it sound like only `CREATE TABLE` and `CREATE MODEL TABLE` are supported.

Reply:
Fixed.